hydra_redis/querying_mechanism.py

```diff
@@ -526,6 +608,9 @@ def help():
           "show <collection_endpoint> properties")
```

Comment:
How is this different from command at line 602?

Reply:
It is different internally if `members` is query then it will check Are these endpoints members loaded or not in Redis?
but if `properties` is a query then the client doesn't check the data it will directly access data from Redis.

Reply:
Can you provide an example?

Reply:
Yeah, I think it is same as line 606. It should be removed from here. I'll update it.